source/stylesheets/_variables.scss

```diff
@@ -0,0 +1,138 @@
+
+//media query size
+$break-large: 960px;
+$break-medium: 769px;
+$break-medium-small: 470px;
+$break-small: 350px;
+
+// color
+$lightest-neutral-color: #F5F7FA; 
+$light-neutral-color: #EFF1F8;  
+$contrast-neutral-color: #FFF3EF; 
+$dark-color: #2D3346;
+$emphasis-color-strong: #CD4722; 
+$emphasis-color-medium: #DE5B34;
+$text-color-dark: #434A53;
+$text-color-lighter: #454E6D;
+$link-underline: #F39779;
+$link-color-secondary: #58606C;
+$visited-link: #4F2478;
+$footer-title-color: #BDC5DD;
+
+//font
+$font-type: 'Montserrat', Arial;
+$base-font-size: 15px;
+$medium-weight: 500;
+$regular-weight: 400;
+$subhead-size: 0.82em;
+$footer-text-size: 0.72em;
+$caps-spacing: 0.14em;
+
+//standard padding 
+$small-padding: .5rem;
+$medium-padding: 1rem; // Used for most 
+$large-padding: 2rem;
+$larger-padding: 2.5rem;
+$extra-large-padding: 5rem;
+
+//line limits
+$line-limit-desktop: 40em;
+$line-limit-mobile: 33em;
+$letter-spacing: 0.025em;
+
+*{
```

Comment:
Move here and below to separate file since they are not variables. Also I don't recommend using `*`. We can use a reset file if you want. Can discuss in person.

Reply:
consider adding in body